README.adoc

```diff
@@ -29,7 +29,7 @@ buildPlugin()
 
 * `repo` (default: `null`  inherit from Multibranch) - custom Git repository to check out
 * `useAci` (default: `false`) - uses link:https://azure.microsoft.com/en-us/services/container-instances/[Azure Container Instances] for build instead of link:https://azure.microsoft.com/en-us/services/virtual-machines/[Azure Virtual Machines].
-* `forceAci` (default: `false`) - forces usage of ACI (see `useAci` above) even on platforms that are not enabled when `useAci` is set to `true` (Windows).
+* `forceAci` (default: `false`) - this parameter is deprecated, please use `useAci` instead.
```

Comment:
why not just remove it?

```suggestion
```

Reply:
It is still in the buildPlugin.groovy script, so I thought it was best to actually notate that it is deprecated instead of just removing it.

Reply:
Now that no plugins are using it though, you could remove it from `buildPlugin.groovy` as well.